src/domain/community/ai-persona/ai.persona.entity.ts

```diff
@@ -3,28 +3,23 @@ import { IAiPersona } from './ai.persona.interface';
 import { AuthorizableEntity } from '@domain/common/entity/authorizable-entity';
 import { AiPersonaInteractionMode } from '@common/enums/ai.persona.interaction.mode';
 import { AiPersonaDataAccessMode } from '@common/enums/ai.persona.data.access.mode';
+import { MINI_TEXT_LENGTH, SMALL_TEXT_LENGTH } from '@common/constants';
 
 @Entity()
 export class AiPersona extends AuthorizableEntity implements IAiPersona {
   // No direct link; this is a generic identifier
-  @Column('varchar', { nullable: false, length: 128 })
+  @Column('varchar', { nullable: false, length: SMALL_TEXT_LENGTH })
   aiPersonaServiceID!: string;
 
   @Column('text', { nullable: true })
-  description = '';
+  description?: string;
```

Comment:
I think description should be mandatory + be set in code to empty string if not provided. Or we require it in the create DTO.

Reply:
A description is usually not mandatory, but we can enforce it in the API layer.
About the column type - the API layer enforces `65568`. We can either match it or standardize it to `DESCRIPTION_LENGTH` of some length.
 What do you think?

Reply:
On reconsideration, we probably want this description to match the definition on the Profile entity - which would make it nullable.